AIPM-Registry/COVERAGE_MAP.md

```
@@ -0,0 +1,119 @@
+
+* * → Langfuse, OpenTelemetry exporters, tracing/metrics/loggin
+
+g
+
+- **Compliance
+
+* * → SOC2/ISO/GDPR/EU AI Act taggin
+
+g
+
+ + data residenc
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)